src/matchmaking.rs

```diff
@@ -311,6 +311,36 @@ impl<Manager> Matchmaking<Manager> {
             false => Err(SteamError::IOFailure),
         }
     }
+
+    /// Gets the data from a lobby chat message after receiving a `LobbyChatMsg_t` callback.
+    ///
+    /// # Parameters
+    /// - `lobby`: The Steam ID of the lobby to get the chat message from.
+    /// - `chat_id`: The index of the chat entry in the lobby.
+    /// - `buffer`: Return the message data by copying it into this buffer. The buffer should be up
+    /// to 4 Kilobytes.
+    ///
+    /// # Returns
+    /// Returns `usize` The number of bytes copied into buffer
+    pub fn get_lobby_chat_entry(&self, lobby: LobbyId, chat_id: i32, buffer: &mut [u8]) -> usize {
+        let steam_user = sys::CSteamID_SteamID_t { m_unAll64Bits: 0 };
+        let mut steam_user = sys::CSteamID {
+            m_steamid: steam_user,
+        };
+        let mut chat_type = steamworks_sys::EChatEntryType::k_EChatEntryTypeInvalid;
+        unsafe {
+            let elements = sys::SteamAPI_ISteamMatchmaking_GetLobbyChatEntry(
```

Comment:
Can just directly return the value, without naming it "elements" first. 

Alternatively, you could reslice the buffer and return it to the user, but it's really a question of taste. 
```rs
unsafe {
    let len= sys::SteamAPI_ISteamMatchmaking_GetLobbyChatEntry(...);
    &buffer[0..len]
}
```

Reply:
Hmm, I think I like returning the resliced buffer, it is much more ergonomic for the consumer.

Also note that I'm not returning `steam_user`, since that information is already available from the `LobbyChatMsg` callback. Even the docs say it is...
> Usually unneeded since it will be the same as [LobbyChatMsg_t](https://partner.steamgames.com/doc/api/ISteamMatchmaking#LobbyChatMsg_t)::m_ulSteamIDUser.

Reply:
addressed